rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfromscreen.adoc

```diff
@@ -0,0 +1,97 @@
+# Interacting with a Screen of a Legacy Application
+
+// number the headings to correspond to the numbered list of steps
+:sectnums:
+
+With the order ID you can now get the corresponding tracking number from the legacy system:
+
+. Run your legacy application.
+. Search for the word `Order-ID` on the screen, click in the edit field next to it, enter the order ID and click *Search*.
+. If there is a result, copy the tracking number to the activity parameter and set the gateway variable `is_orderid_found` to true. If there is no result, set the gateway variable `is_orderid_found` to false.
+
+image::usecase-legacysystem-screeninteraction.png["The Workflow contains the steps in form of Action Steps. Some of the Action Steps are nested."]
+
+## Run a Legacy Desktop Application
+
+To run the legacy desktop order tracking application, you select the path to the executable file in the *Run program* Action Step. You set the start mode of the application to `Maximized` to open the application in a maximized window. This ensures that the elements of the UI are always in the same position on the screen, which is important because you later use Action Steps that work with screen coordinates. You select `Wait for Window to appear` and specify the window title to ensure that the following Action Steps won't return an error because the window isn't there yet.
+
+image::usecase-legacysystem-screeninteraction-run.png[]
+
+## Enter Text Into a Screen Element and Click a Button
+
+To search for the tracking number, you have to enter the order ID in the search field and click the search button.
```

Comment:
```suggestion
To search for the tracking number, enter the order ID in the search field and click the search button.
```

Reply:
Without the bot , the user would have to enter the order ID in the search field on the terminal screen to get the tracking number. We will now see, how to explain this to a bot.
Change nevertheless?

Reply:
[There a two viewpoints from which this text is written: What a real user of the application must see and do to achieve the goal and what the RPA Developer must see and do, to get the bot to achieve the goal.]